crates/cxx-qt-lib/build.rs

```diff
@@ -19,7 +19,8 @@ fn main() {
         qt_modules.push("Qml".to_owned());
     }
 
```

Comment:
Note that we don't want merge commits in `main` so please prefer using rebase rather than merging from `main`.

Reply:
Sorry, I realize now that it was not my call to merge/rebase.

Reply:
Thanks! Also note if you aren't used to the rebase model you can then rearrange / squash commits together etc (just watch our for merge conflicts :-) ). So one could keep amending the same commit rather than adding new ones for each feedback comment etc, so then each commit can have individual meaning and be atomic in it's purpose.

Reply:
Thank for your explanations, I'll do my best to commit this way :)